tests/core/Jalasoft.TeamUp.Projects.Core.Tests/Validators/InvitationValidatorTests.cs

```diff
@@ -0,0 +1,132 @@
+﻿namespace Jalasoft.TeamUp.Projects.Core.Tests.Validators
+{
+    using System;
+    using System.Collections.Generic;
+    using FluentValidation.TestHelper;
+    using Jalasoft.TeamUp.Projects.Core.Validators;
+    using Jalasoft.TeamUp.Projects.Models;
+    using Xunit;
+
+    public class InvitationValidatorTests
+    {
+        private readonly InvitationValidator validator;
+
+        public InvitationValidatorTests()
+        {
+            this.validator = new InvitationValidator();
+        }
+
+        public static IEnumerable<object[]> Invitations =>
+        new List<object[]>
+        {
+            new object[] { new Invitation { Status = "Invited" } },
+            new object[] { new Invitation { Status = "Accepted" } },
+            new object[] { new Invitation { Status = "Rejected" } }
+        };
+
+        [Fact]
+        public void InvitationValidator_Valid_Id_Success()
```

Comment:
https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices please follow the naming convention adopted by the team in the other unit tests